src/hooks/useQueryParamFilters.js

```diff
@@ -24,20 +24,26 @@ export const useQueryParamFilters = (filters, fetchData) => {
   }
 
 
-  const addQueryParamFilter = (newQueryParam) => {
-    const keyExists = queryParamsFilter.some(
-      (param) => param.key === newQueryParam.key
-    );
-
-    const updatedQueryParamsFilter = keyExists
-      ? queryParamsFilter.map((param) =>
-        param.key === newQueryParam.key ? {
-          ...param,
-          value: newQueryParam.value 
-        } : param
+  const addQueryParamFilter = (newQueryParams) => {
```

Comment:
lo cambié para q reciba el from y to todo junto, sino andaba mal.

Reply:
Mmm.. me parece muy especifico, y para esto me parece mejor crear otra funcion que añada un array en vez de tocar la funcionalidad ya existente. Entonces, tendriamos los dos metodos: uno para añadir una key individual y otro para un array. En el caso de usar datepíckers agarrarias el de subir un array